tests/folium_tests.py

```diff
@@ -510,6 +515,7 @@ def test_image_overlay(self):
         self.map.image_overlay(data, filename=image_url)
         assert image_rendered in self.map.template_vars['image_layers']
 
+        # test inline png
```

Comment:
I know it is a minor thing, but can you start the sentences with a Capital letter and end with a period**.** Also, remove the white space below.